src/lib/graphql/resolvers/Query/index.ts

```diff
@@ -0,0 +1 @@
+export const wizard = (): string => 'nice!';
```

Comment:
If you return something other than a string, does that cause type errors? Just making sure all the types are hooked up appropriately.

Reply:
Yes, changing the return type results in type errors in the code that composes the gql schema.